app/src/main/res/values/settings_keys.xml

```diff
@@ -245,6 +245,7 @@
     <string name="crash_the_app_key">crash_the_app_key</string>
     <string name="show_error_snackbar_key">show_error_snackbar_key</string>
     <string name="create_error_notification_key">create_error_notification_key</string>
+    <string name="settings_layout_redesign_key">settings_layout_redesign_key</string>
```

Comment:
Once we switch completely to Compose we should also migrate away from SharedPreferences, but rather use Kotlin DataStore. We'll not need to have settings keys in a strings file anymore (since there won't be XML files), which will also simplify much of the code by allowing to refer to string constants in code.

Reply:
Sounds great. I think this should be done independently of the Compose/Settings UI redesign. Created #11017 for this.